.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  extends: 'standard',
+  ignorePatterns: [
+    '!.*',
+    'dist/**/*',
+    'node_modules',
+    'node_modules/.*',
```

Comment:
I think this is unnnecessary as ESLint already ignores `node_modules` ([buried in this page of their doc](https://eslint.org/docs/latest/user-guide/configuring/ignoring-code)):

>In addition to any patterns in the .eslintignore file, ESLint always follows a couple of implicit ignore rules even if the --no-ignore flag is passed. The implicit rules are as follows:
>
>  -  node_modules/ is ignored.
>  -  dot-files (except for .eslintrc.*), as well as dot-folders and their contents, are ignored.

Reply:
Ahh sorry, I should explain

See a bit further up I've added `'!.*'`?

It's to override ESLint's "ignore dot-files by default" logic so we can lint files such as:

```
.eslintrc.js
.lintstagedrc.js
.babelrc.js
```

But then sadly it seems to then include "dot files inside node_modules" which made this combo necessary:

```
'node_modules',
'node_modules/.*',
```

Glad you spotted it though, I love a good code review 😊